City.h

```diff
@@ -0,0 +1,57 @@
+#include "Area.h"
+class City{
+    private:
+    vector<Area> areas;
+    string nameCity;
+    public:
+    void printCity(){
+        cout << "City:" << nameCity << endl;
+    }
+
+    City(string tmpName){
+        nameCity = tmpName;
+    }
+
+    void addArea(Area  area){
+        areas.push_back(area);
+    }
+
+    void printAllArea()
+    {
+        for (int i = 0; i < areas.size(); i++)
+        {
+            cout << i+1 <<".";
+            areas.at(i).printArea();
+        }
+
+        cout << "Count of area: " << areas.size() << endl;
+    }
+
+};
+
+
+
+class Contry{
+    private:
+    vector<City> cities;
+    string name;
+    public:
+    Contry(string tmpName){
+        name = tmpName;
+    }
+    void addCity(City name){
```

Comment:
The name of method shoud be in uppercase.

Reply:
Can be as it is.